### ISSUE TITLE
Sony A5000 and 60FPS

### ISSUE THREAD

**Issue by FrozenGamesYT** [ma1co/Sony-PMCA-RE#430]
is there a way to bruteforce 1080p60 instead of 1080i60 on the avchd format or in any other way?

(for obs streaming over hdmi cable to hdmi grabber [camlink for example])

**Comment by SammyCola** [comment]
Not sure if there is a way for recording to the memory card, but if you just using the live preview through HDMI out, in Settings Section 2, you can set 1080i/1080p for the HDMI Resolution, and it appears at first glance to be the proper resolution.

**Comment by FrozenGamesYT** [comment]
> Not sure if there is a way for recording to the memory card, but if you just using the live preview through HDMI out, in Settings Section 2, you can set 1080i/1080p for the HDMI Resolution, and it appears at first glance to be the proper resolution.

yeah but the live preview seems to be 60i cuz of the settings i believe, could be wrong

**Comment by SammyCola** [comment]
Live preview through HDMI correct? When i set 1080i for HDMI, i get 1080i60. When setting to 1080p, i see 1080p60.

**Comment by FrozenGamesYT** [comment]
> Live preview through HDMI correct? When i set 1080i for HDMI, i get 1080i60. When setting to 1080p, i see 1080p60.

Correct, how do i check if my output is really 1080p60? In the settings i have 1080p set up with avchd and 60i because there isn't 60p option

**Comment by SammyCola** [comment]
Check your HDMI display for what resolution it's getting. If it's a computer monitor, usually it'll tell you in the settings for the monitor. If it's a TV, varies, on my Sony OLED it's pressing the "Display" button on the remote, and it'll list the HDMI format (1080i and 1080p when HDMI Resolution is set).

On the camera, these settings are for recordings saved to the SD card (they don't affect HDMI from what I see, and are named straightforward):
Camera Icon > 2 > File Format
Camera Icon > 2 > Record Setting

On the camera, these settings are for the HDMI out:
Settings Icon > 2 > HDMI Resolution (this is what sets the HDMI output, use 1080p).
